portfolio/src/main/webapp/script.js

```diff
@@ -84,4 +84,10 @@ function deleteComment(comment) {
   const params = new URLSearchParams();
   params.append('id', comment.id);
   fetch('/delete-data', {method: 'POST', body: params});
-}
\ No newline at end of file
+}
+
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: -33.865, lng: 151.209}, zoom: 12});
```

Comment:
Can we have a constant explaining the location?

Reply:
yes will change that

Reply:
Done!